app/components/primer/base_button.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
```

Comment:
Do you think we should be exposing this component as a public API? IMO we should keep it private, at least to start.

Reply:
I'm fine keeping it private, that's why I didn't add it to the docs sidebar 😄

Reply:
Ah, cool. 👍🏻